.github/workflows/release_candidate.yml

```diff
@@ -68,6 +68,8 @@ jobs:
         run: |
           if ${{ contains(steps.current_version.outputs.current_version, 'rc') }} ; then
             npm version prerelease --preid=rc --no-git-tag-version
+          elif ${{ contains(steps.current_version.outputs.current_version, 'next') && inputs.version == 'prepatch' }} ; then
+            npm version prerelease --preid=rc --no-git-tag-version
```

Comment:
if the current version is a `next` prerelease, running `prepatch` will increment the patch version one more - to avoid that, we run `prerelease --preid=rc` instead which will just change `-next` to `-rc`